### PR TITLE
Add "patbenatar.advanced-new-file" extension

### PR DESCRIPTION
**Pull request by rafiazman**
A better implementation of adding new files, in my humble opinion.

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -1192,6 +1192,12 @@
       "download": "https://github.com/paiqo/Databricks-VSCode/releases/download/v0.6.0/databricks-vscode-0.6.0.vsix",
       "version": "0.6.0"
     },
+    {
+      "id": "patbenatar.advanced-new-file",
+      "repository": "https://github.com/patbenatar/vscode-advanced-new-file",
+      "version": "1.2.2",
+      "checkout": "master"
```
Comment (jankeromnes) [inline extensions.json:1199]: This repository has no GitHub Releases or dedicated release tag, so there is no need to try and pin or checkout a specific version.

The best we can do here is publish straight from the default branch.

```suggestion
      "repository": "https://github.com/patbenatar/vscode-advanced-new-file"
```

**Review by jankeromnes — APPROVED** [review]
Thanks @rafiazman! 👍 

One minor adjustment and this is good to go. 🚀

**Comment by jankeromnes** [comment]
Update: Auto-publishing worked! 🎉 https://open-vsx.org/extension/patbenatar/advanced-new-file

@rafiazman Please kindly confirm that the extension is working as intended.

**Comment by rafiazman** [comment]
Works great, thank you!

**Comment by rafiazman** [comment]
@jankeromnes The original developer of the extension has published to Open VSX (see: https://github.com/patbenatar/vscode-advanced-new-file/issues/87).

Hence, I think its best for this to be removed to eliminate confusion.

**Comment by patbenatar** [comment]
Hey all- sorry it took me some time to publish. I published under a different namespace because patbenatar namespace was already created in this PR. Can I be given access to publish to that namespace?

**Pull request merged**